psalm.xml

```diff
@@ -24,7 +24,6 @@
     </plugins>
     <forbiddenFunctions>
         <function name="dd"/>
-        <function name="dump"/>
```

Comment:
I remove it for now as we actually have some dumps in the code now 🚀 Should be added later.

Reply:
Or, I will just remove those dumps 🔥